benchmark-service/benchmark/benchmark.py

```diff
@@ -1,19 +1,28 @@
 import os
 from pathlib import Path
 from typing import Any, AsyncIterator, Dict
-
+import asyncio
+import aiohttp
+import aiohttp_session
 import aiohttp_jinja2
 import jinja2
 from aiohttp import web
 import logging
+from gear import setup_aiohttp_session, web_authenticated_developers_only, rest_authenticated_developers_only, AccessLogger, configure_logging
+from hailtop.config import get_deploy_config
+from hailtop.tls import get_server_ssl_context
+from web_common import setup_aiohttp_jinja2, setup_common_static_routes
 
 
 router = web.RouteTableDef()
 logging.basicConfig(level=logging.DEBUG)
+deploy_config = get_deploy_config()
+#configure_logging()
```

Comment:
Can you uncomment the `#configure_logging()` and move it before the line with the router?